src/main/java/it/pagopa/ecommerce/payment/methods/controller/v2/PaymentMethodsController.java

```diff
@@ -74,4 +83,27 @@ public ResponseEntity<ProblemJsonDto> errorHandler(RuntimeException exception) {
         }
     }
 
+    @Warmup
+    public void calculateFeesWarmupMethod() {
+        CalculateFeeRequestDto request = new CalculateFeeRequestDto()
+                .bin("BIN_TEST")
+                .touchpoint("CHECKOUT")
+                .addIdPspListItem("string")
+                .idPspList(new ArrayList<>(List.of("first", "second")))
+                .isAllCCP(false)
+                .paymentNotices(Collections.emptyList());
```

Comment:
```suggestion
          CalculateFeeRequestDto request = new CalculateFeeRequestDto()
                .touchpoint("CHECKOUT")
                .isAllCCP(false)
                .addPaymentNoticesItem(
                        new PaymentNoticeDto()
                                .paymentAmount(100L)
                                .primaryCreditorInstitution("77777777777")
                                .addTransferListItem(
                                        new TransferListItemDto()
                                                .digitalStamp(false)
                                                .creditorInstitution("77777777777")
                                )

                );
```
The input request is invalid (warmup call lead to a 400 BAD request.
Change it as follow to produce a valid request